src/components/PetListingPreview.js

```diff
@@ -4,17 +4,20 @@ import catSteps from '../images/paw-print.jpg'
 import PetListing from './PetListing';
 import PetCard from './PetCard';
 
-const PetListingPreview = ({pets}) => (
-    <Wrapper>
-        <Title>
-            Pets Available for Adoption
-        </Title>
-        <ListingWrapper>
-            <PetListing pets={pets} />
-            <MeetMoreButton name={"Meet more friends"} img={catSteps}/>
-        </ListingWrapper>
-    </Wrapper>
-)
+const PetListingPreview = ({pets}) => {
+    let imagesArr = [catSteps];
+    return(
+        <Wrapper>
+            <Title>
+                Pets Available for Adoption
+            </Title>
+            <ListingWrapper>
+                <PetListing pets={pets} fromPreview/>
+                <MeetMoreButton name={"Meet more friends"} imgs={imagesArr}/>
```

Comment:
name={"Meet more friends"}
you're passing what JS type exactly?

Reply:
fixed